src/tiled/changetileprobability.cpp

```diff
@@ -29,11 +29,11 @@ namespace Tiled {
 namespace Internal {
 
 ChangeTileProbability::ChangeTileProbability(MapDocument *mapDocument,
-                                             Tile *tile,
-                                             float probability)
+                                             QList<Tile*> tiles,
+                                             QList<float> probabilities)
```

Comment:
These containers should be passed as `const ... &`. It's only saving some reference counting, but still.